extensions/github/src/hooks/useIssues.ts

```diff
@@ -0,0 +1,52 @@
+import { useCachedPromise } from "@raycast/utils";
```

Comment:
I don't see this hook used anywhere in the extension. Should it be removed?

Reply:
It is used in line 11